sdk/Lusid.Sdk.Tests/LusidApiFactoryTests.cs

```diff
@@ -224,11 +224,11 @@ public void Multi_Threaded_ApiFactory_Tasks(int quoteCount, int threadCount)
                 //api.Configuration.ApiClient.Dispose();
 
                 // Option 2: force all finalizers to run
-                if (i % 100 == 0)
-                {
-                    GC.Collect();
-                    GC.WaitForPendingFinalizers();
-                }
+                // if (i % 100 == 0)
+                // {
+                //     GC.Collect();
+                //     GC.WaitForPendingFinalizers();
+                // }
```

Comment:
What is this commented out section for?

Reply:
This tested was added for the previous version of the SDK, I believe the issue goes away with the upgrade (tested and couldn't reproduce), i left it in just in case it does reappear.